sections/01_abstract.md

```diff
@@ -3,3 +3,14 @@
 Offspring of the [Deep Review](https://github.com/greenelab/deep-review).
 We cite the Deep Review like [@doi:10.1101/142760] or [@tag:deep_review].
 The source repository is [@url:https://github.com/greenelab/manubot-rootstock].
+
+Equations can be integrated and labeled like this `{{ '{{ \'{#eq:label}\' }}' }}` and then
+referenced using `{@eq:label}` like this for Equation @eq:line.
+
+$$ y = mx + b $$ {{ '{#eq:line}' }}
+
+Images can be included and labeled like `{{ '{{ \'{#fig:label}\' }}' }}` and then
+referenced using `{@fig:label}` like this for Figure @fig:lotus.
+
+![An example image.](./images/lotus.jpg){{ '{#fig:lotus}' }}
```

Comment:
Does `images/lotus.jpg` work instead of `./images/lotus.jpg`? Assuming those are the same?

If I'm understanding correctly, this relative path is just passed through... no modifications?